applications/nrf_desktop/src/hw_interface/leds.c

```diff
@@ -75,6 +74,15 @@ static void work_handler(struct k_work *work)
 		if (led->effect_step == led->effect->step_count) {
 			if (led->effect->loop_forever) {
 				led->effect_step = 0;
+			} else {
+				LOG_INF("Set next step");
```

Comment:
this log is not valid anymore
you can remove it as we will see the information on the event manager logs